backend/node/base.py

```diff
@@ -18,6 +19,7 @@ def __init__(
         model: str,
         config: dict,
         leader_receipt: Optional[Receipt] = None,
+        msg_handler: MessageHandler = None,
```

Comment:
The GenVM is not handling the possibility of `msg_handler == None`, so I would make it a required parameter